src/utils/encode.ts

```diff
@@ -183,15 +187,14 @@ function padString(str: string, length: number, left: boolean, padding = STRING_
  * @param {number} length The target length for the padded string.
  * @param {string} [padding='0'] The string to use for padding. Defaults to '0'.
  * @returns {string} The padded string.
- *
- * @example
+ * * @example
```

Comment:
```suggestion
 * @example
```

I think examples are without this extra *